configuration/controller.cfg

```diff
@@ -353,3 +354,12 @@ LUNA_NETWORK_NAME=cluster
 LUNA_DHCP_RANGE_START=10.30.128.1
 LUNA_DHCP_RANGE_END=10.30.255.200
 
+
+#-----------------------------------------------------------
+# rsyslog
+#-----------------------------------------------------------
+
+RSYSLOG_MESSAGES_PATH="/var/log/cluster-messages"
+RSYSLOG_CLIENT_NETWORK=${LUNA_NETWORK}
```

Comment:
Rsyslog can be installed on clusters that don't include Luna, so the Luna configuration might not be here. Is there any way that rsyslog can be configured only with the IPs of the controllers?